examples/goext_example/todo/entry.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2017 NTT Innovation Institute, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package todo
+
+// Entry is a single todo entry
+type Entry struct {
```

Comment:
yes, there is a converter:
https://github.com/Zimnx/YamlSchemaToGoStruct
which does this task automatically. Currently it is not integrated into gohan cli to avoid too large PR (a few thousand of extra lines). Of course, there are plans to integrate it later. In this version the code is written by hand.